warehouse/ingest-core/src/main/java/datawave/ingest/table/aggregator/GlobalIndexUidAggregator.java

```diff
@@ -74,15 +73,26 @@ protected boolean isSeenIgnore() {
     }
 
     /**
-     * We want timestamps in the global index to be combined separately. Normally all of the timestamps on the global index are the day at 00:00:00. However we
-     * could have a composite timestamp (@see CompositeTimestamp) in which case we want separate entries.
+     * Normally this class will return the most recent key associated with the combined values. We need this key to have a truncated timestamp.
+     *
+     * @return The top key with a truncated timestamp.
+     */
+    @Override
+    public Key getTopKey() {
```

Comment:
I don't believe this actually makes a difference as the PropogatingIterator is using the key from the underlying source.